pkg/appliance/functions.go

```diff
@@ -596,6 +584,86 @@ func applyApplianceFilter(appliances []openapi.Appliance, filter map[string]stri
 	return filteredAppliances, nil
 }
 
+func reverse[T openapi.Appliance | openapi.StatsAppliancesListAllOfData](a []T) []T {
+	rev := []T{}
+	for i := len(a) - 1; i >= 0; i-- {
+		rev = append(rev, a[i])
+	}
+	return rev
+}
```

Comment:
just nitpicking now, but we can make it even more generic
```suggestion
func reverse[S ~[]T, T any](items S) S {
	if len(items) <= 1 {
		return items
	}
	result := make([]T, 0, len(items))
	for i := len(items) - 1; i >= 0; i-- {
		result = append(result, items[i])
	}
	return result
}
```

Reply:
Sure. Still not used to the generics, but this could definitely be more generic.